Ugee_G5/index.md

```diff
@@ -0,0 +1,19 @@
+---
+VID: "28bd"
+PID: "0074"
+vendor: Ugee
+product: G5
+image: Ugee_G5
+report_rate: 220
+working_area:
+    width: 9
+    height: 6
+    resolution: 5080
+pen:
+    pressure_levels: 2048
+frame_controls: 5 buttons
```

Comment:
Thanks for adding the buttons, but you lost the wheel now :) I'll fix this and merge.